ci-operator/templates/openshift/installer/cluster-launch-installer-metal-e2e.yaml

```diff
@@ -428,8 +428,8 @@ objects:
         cp -r /var/lib/openshift-install/upi/metal/* /tmp/artifacts/terraform/
         cp /bin/terraform-provider-matchbox /tmp/artifacts/terraform/
 
-        PXE_INITRD_URL="$(jq -r '.baseURI + .images["initramfs"].path' /var/lib/openshift-install/rhcos.json)"
```

Comment:
I believe `rhcos.json` is to be deprecated. Should use arch specific file such as `rhcos-amd64.json`

Reply:
Ultimately it will be replaced with https://github.com/openshift/enhancements/pull/201

Using `rhcos-amd64.json` makes sense to me, though it seems orthogonal to this PR and could be done as a followup.